### DISCUSSION TITLE
How to import a batch of Sigma rules into Playbook (Standalone Airgapped deployment)

### DISCUSSION THREAD

**Discussion by kdice101** [Security-Onion-Solutions/securityonion#9177] [Unsupported Versions] [answered]
Hello as the title says, trying to import a mass of Sigma rules into Playbook outside of the Web Gui.  The adding a play using the insert .csv function seems way too slow and cumbersome. 

1. Is it possible to import Sigma rules via the CLI through the so-manager? IF so, what is the order of operations? We were experimenting with just adding the YAMLs directly into the /opt/so/rules/elastalert/playbook/ directory but that didnt seem to work. 

2.What directory is Playbook referencing when creating rules? we assumed it was the /opt/so/rules/elastalert/playbook directory, so we deleted all the Plays from the Web front end and renamed the /opt/so/rules/elastalert/playbook directory/replaced it with a empty ..../playbook directory, reran so-playbook-ruleupdate and all of the  prebuilt Plays we cleared out of the Web front end were recreated.

Any tips would greatly be appreciated.

**Comment by defensivedepth** [comment, marked as answer]
What version of Security Onion are you on?

In version 2.3.80, we added the following capability:

Drop the individual sigma rule yaml files under `/opt/so/conf/soctopus/sigma-import`  & run `sudo so-playbook-import `.

It doesnt do dupe checking so if you run it twice, it will add dupes. Also, you can run `so-playbook-import True` to create the Plays & make them active
